templates/keystoneapi/config/keystone.conf

```diff
@@ -4,8 +4,12 @@ use_stderr=true
 [cache]
 {{if .memcachedTLS}}
 backend = dogpile.cache.pymemcache
+enable_retry_client = true
+retry_attempts = 2
+retry_delay = 0
 {{else}}
 backend = dogpile.cache.memcached
+memcache_dead_retry = 10
```

Comment:
from puppet tripleo I can see we used to have it a 60 for all services

Reply:
We can't really compare vs tripleo unfortunately. There we have static ips (controllers ips do not change) while here every time we get a new pod the ip changes, so we need to react faster imho.